dandi/utils.py

```diff
@@ -347,9 +350,17 @@ def yaml_dump(rec):
     to assure proper formatting on versions of pyyaml before
     5.1: https://github.com/yaml/pyyaml/pull/256
     """
-    import yaml
+    yaml = ruamel.yaml.YAML(typ="safe")
+    yaml.default_flow_style = False
+    out = io.StringIO()
+    yaml.dump(rec, out)
+    return out.getvalue()
+
 
-    return yaml.safe_dump(rec, default_flow_style=False)
+def yaml_load(f, typ=None):
+    # `f` can be either a string or a file-like object.
+    # `typ` defaults to "rt" (round-trip)
```

Comment:
just not to delay - now that it is in `utils`, it better come with a proper docstring.  We (loosely, might need fixes) following numpy docstring style through the codebase.

Reply:
Done.